src/core/utils/misc.cpp

```diff
@@ -315,30 +315,58 @@ QString Utils::Misc::pythonVersionComplete() {
     return version;
 }
 
+QString Utils::Misc::unitString(Utils::Misc::SizeUnit unit)
+{
+    return QCoreApplication::translate("misc",
+         units[static_cast<int>(unit)].source, units[static_cast<int>(unit)].comment);
+}
+
 // return best userfriendly storage unit (B, KiB, MiB, GiB, TiB)
 // use Binary prefix standards from IEC 60027-2
 // see http://en.wikipedia.org/wiki/Kilobyte
 // value must be given in bytes
 // to send numbers instead of strings with suffixes
-QString Utils::Misc::friendlyUnit(qreal val, bool is_speed)
+
+bool Utils::Misc::friendlyUnit(qreal rawVal, qreal &val, Utils::Misc::SizeUnit &unit)
```

Comment:
Technically, Yes, but not in meaning. The meaning of overloaded functions is to do the same thing for different params. For example:
```c++
int add(int a, int b) { return a + b; }
double add(double a, double b) { return a + b; }
```
these are overloaded functions (by definition and by meaning).
In your case one function reduces the number up to a certain order, and another converts a number to a string in some way.